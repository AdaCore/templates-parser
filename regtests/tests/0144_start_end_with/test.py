from test_support import *

run('testme', ["testme144.tmplt"])
