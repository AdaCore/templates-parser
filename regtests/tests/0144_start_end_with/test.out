
Test START_WITH() routine.

TRUE  = TRUE
TRUE  = TRUE
TRUE  = TRUE
TRUE  = TRUE
FALSE = FALSE
FALSE = FALSE
FALSE = FALSE
FALSE = FALSE
FALSE = FALSE

Test END_WITH() routine.

TRUE  = TRUE
TRUE  = TRUE
TRUE  = TRUE
TRUE  = TRUE
FALSE = FALSE
FALSE = FALSE
FALSE = FALSE
FALSE = FALSE
FALSE = FALSE
